web-app/eslint.config.mjs

```diff
@@ -31,6 +31,7 @@ const eslintConfig = [
       "import/newline-after-import": "error",
       "import/no-duplicates": "error",
       "unused-imports/no-unused-imports": "error",
+      "@typescript-eslint/prefer-nullish-coalescing": "error",
```

Comment:
We have strictNullCheck enabled implicitly, right? As far as I see yes, but maybe you can double-check

Reply:
Ok as far as I see it is by default enabled as long as we have https://www.typescriptlang.org/tsconfig/#strict maybe we should still add it in case this ever changes?

Reply:
Please Google this yourself in the future if you're unsure. It doesn't help if I search for it and then paste the response here. You can refer to this link: https://www.typescriptlang.org/tsconfig/#strict.

I checked it based on this reference:
> This rule will not work as expected if [strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) is not enabled.

If you had Googled it, you would have seen that it is enabled. Now, I need to write this comment and request a review again.

Reply:
Ok I see the second comment with the context was not added directly, very strange. Should we add the strictNull checks regardless as mentinoed? In order to do be explicit

Reply:
Make a call. I don't care.

Reply:
added in https://github.com/masumi-network/sokosumi/pull/126/commits/0a110609f7cb88be5882a472e348211db999964c